CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.0]
+
+### Changed
+
+- **BREAKING** Update @metamask/keyring-api to version 6.0.0 ([#275](https://github.com/MetaMask/eth-snap-keyring/pull/275))
```

Comment:
- `**BREAKING**` -> `**BREAKING**:`
- Update -> Bump? (we should have change the PR title probably 🙈), so I'm not sure about this one, WDYT?

Reply:
Yea bump is more consistent